src/test/resources/goal-output/commons-math3-3.6.1/org.apache.commons.math3.fraction.BigFraction_goal.json

```diff
@@ -54,7 +54,7 @@
           "isArray": false
         },
         "comment": "if the denominator is zero.",
-        "condition": "den.signum()==0"
+        "condition": "args[0].signum()==0"
```

Comment:
I agree in the number of the parameter. For the comparison between the variable and zero, I'd rather use a function that gives me the value of the object, like signum or abs, but, of course, this option is more complex for Toradocu.